File: src/test/scala/cern/colt/matrix/tlong/impl/SparseCCLongMatrix2DViewTest.scala
package cern.colt.matrix.tlong.impl

//remove if not needed
import scala.collection.JavaConversions._

class SparseCCLongMatrix2DViewTest(arg0: String) extends SparseCCLongMatrix2DTest(arg0) {

  protected def createMatrices() {
    A = new SparseCCLongMatrix2D(NCOLUMNS, NROWS).viewDice()
    B = new SparseCCLongMatrix2D(NCOLUMNS, NROWS).viewDice()
    Bt = new SparseCCLongMatrix2D(NROWS, NCOLUMNS).viewDice()
  }
}

File: src/test/java/cernx/colt/matrix/tlong/impl/SparseCCLongMatrix2DViewTest.java
package cernx.colt.matrix.tlong.impl;

public class SparseCCLongMatrix2DViewTest extends SparseCCLongMatrix2DTest {

    public SparseCCLongMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new SparseCCLongMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new SparseCCLongMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new SparseCCLongMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }

}
